build-image/Dockerfile

```diff
@@ -19,7 +19,7 @@ FROM alpine:3.16 as docker
 RUN apk add --no-cache docker-cli docker-cli-buildx
 
 # Dependency: Go and Go dependencies
-FROM golang:1.18-bullseye as golang
+FROM golang:1.19.2-bullseye as golang
```

Comment:
I thought about changing this to just `1.19` but since we don't automatically update the build image, it's better for us to be explicit about which minor version we're using so we can upgrade it when necessary.